wave_lang/kernel/wave/utils/run_utils.py

```diff
@@ -144,5 +144,6 @@ def set_default_run_config(options: WaveCompileOptions) -> WaveCompileOptions:
         options.use_scheduling_barriers = bool(int(enable_scheduling_barriers))
 
     options.backend = "rocm"
+    options.device = "hip"
```

Comment:
`rocm` backend is already implies `hip` device, don't see much reason to keep it in options